spec/server/serverApp.test.js

```diff
@@ -0,0 +1,23 @@
+const request = require('supertest');
+const app = require('../../server/serverApp.js');
+
+describe('Server endpoint tests', () => {
```

Comment:
Awesome use of supertest! From what I researched, this is correct way to test your server endpoints.